mysqlnd_azure.c

```diff
@@ -366,13 +406,19 @@ MYSQLND_METHOD(mysqlnd_azure_data, connect)(MYSQLND_CONN_DATA ** pconn,
             enum_func_status ret = FAIL;
             MYSQLND* redirect_conneHandle = mysqlnd_init(MYSQLND_CLIENT_KNOWS_RSET_COPY_DATA, conn->persistent); //init MYSQLND but only need only MYSQLND_CONN_DATA here
             if(!redirect_conneHandle) {
+              if (MYSQLND_AZURE_G(logLevel) > 0) {
+                AZURE_LOG("DEBUG", "Cannot connet with refirection information, redirect connection handshake failed");
```

Comment:
Here the handshake for redirection is not started yet. This step is to check whether the object is initialized correctly using mysqlnd_init, the possible fail reason can be oom. So may be change to " Init redirect_conneHandle failed" ? Which is same as debug formatted log